### DISCUSSION TITLE
Some URLs cannot be opened

### DISCUSSION THREAD

**Discussion by ghost** [cefsharp/CefSharp#3545] [General]
For example:
https://www.bet365.com

Version: CefSharp.WinForms 86.0.241.0

thx.

**Comment by amaitland** [comment]
There's already a discussion about bet365 at https://github.com/cefsharp/CefSharp/discussions/3377

Bet365 are deliberately blocking embedded browsers, this is not something we can control.

**Reply by ghost** [reply]
Thank you very much. ^_^
